app/views/RoomView/RightButtons.tsx

```diff
@@ -346,6 +382,12 @@ class RightButtonsContainer extends Component<IRightButtonsProps, IRigthButtonsS
 		}
 		return (
 			<HeaderButton.Container>
+				<HeaderButton.Item
```

Comment:
This looks wrong.
Are we really going to display the troubleshooting on RoomView even if there's nothing wrong with push?

Reply:
No. It's expected the user have direct access to the notification preference through RoomHeader.

@jsdgdo said this to me
> for the Room header yes, the icon shows up black when nothing's wrong, for direct access to the notification preferences, changing it to red when something's wrong.

Reply:
I was confused when stating this, it was my mistake. The black icon should only appear when notifications are disabled for the room, it says so on the Figma file, I thought about it the other way initially and I changed my mind after, did not tell Reinaldo anymore.